templates/CRM/Price/Form/ParticipantCount.tpl

```diff
@@ -1,191 +0,0 @@
-{*
- +--------------------------------------------------------------------+
- | Copyright CiviCRM LLC. All rights reserved.                        |
- |                                                                    |
- | This work is published under the GNU AGPLv3 license with some      |
- | permitted exceptions and without any warranty. For full license    |
- | and copyright information, see https://civicrm.org/licensing       |
- +--------------------------------------------------------------------+
-*}
-
-{if $priceSet && $allowGroupOnWaitlist}
```

Comment:
Civi is full of fun stuff to find, huh?

I'm not immediately seeing what this smarty `{if}` was replaced by? There's `if ($this->_values['event']['is_multiple_registrations'] && isset($this->_values['event']['max_participants']))` but it doesn't look like the same thing?

Also I see two other templates that will still try to include this tpl?

Reply:
Hmm - I assumed that was the only screen that would have any reason to include this .tpl, and I relied on the change I made to the PHP file that determined whether to load this .tpl at all.  I'll go back and review the other files that include this.